### DISCUSSION TITLE
How to style the selection list from combo-box-text widget in scss?

### DISCUSSION THREAD

**Discussion by MartyMcFlyInTheSky** [elkowar/eww#1162] [Q&A]
I want to use the combo-box-text widget to select my wifi connection. I have made this to test:

```
        (combo-box-text
            :class "combo-box"
            :items "[Hello, asdf, Element2]"
            :onchange "[notify-send 1234, notify-send 5678]"))
```

I can format the "selected"-field with that class "combo-box" in my scss. However I can't find a way to also format the selection list that appears when clicking on the selected item to re-select. What would the appropriate class be named?

Also is the list syntax correct or can I also use children here? (sorry newb to yuck :X)

**Comment by HeathRiewe** [comment]
Hello @MartyMcFlyInTheSky , I have not been particularly successful with configuring the scss for the combo-box-text.  Can you share your file?  I know that it isn't the point of your request, but...
(combo-box-text
      :class "combo-box"
      :items "[Hello, asdf, Element2]"
      :onchange "notify-send 'Hello' 'Hello, {}'"
    )

will actually get the notify-send to work.  (you might be using a different one then me).  Note that the {} expands to the selected text.

I do not get a button on combo-box-text unless scss crashes and the defaults are in place.

**Reply by MartyMcFlyInTheSky** [reply]
Unfortunately right now I don't have a working setup for combo box text, but maybe I'll get back to it later. Thanks for sharing anyway!
